### DISCUSSION TITLE
已解决。

### DISCUSSION THREAD

**Discussion by bhu1987** [getlantern/lantern-forum-cn#534] [General]

**Comment by MaxMKMax** [comment]
@bhu1987 您好. 我已经帮您提交问题上去了. 这两天会给您发激活码. 请查看邮箱或Github. 谢谢!

**Comment by MaxMKMax** [comment]
@bhu1987 您好. 可以的, 我帮您要求添加 3 个月. 也会帮您反馈这个问题.

**Comment by bhu1987** [comment]


**Reply by MaxMKMax** [reply]
@bhu1987 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by MaxMKMax** [comment]
@bhu1987 嗯嗯. 好的. 下周帮您恢复帐号. 很抱歉负责人这几天不在线. 下周回他来后帮您搞定.

**Reply by MaxMKMax** [reply]
@bhu1987 您好. 非常抱歉影响了您的体验感. 我已经帮您提交退款申请了.

**Reply by MaxMKMax** [reply]
@bhu1987 您好. 请您再次提供我您的账号邮箱. 我帮您检查看. 退款的问题是支付部门负责的. 我来帮您问问看进展. 非常抱歉让您久等了!

**Reply by MaxMKMax** [reply]
@bhu1987 您稍等. 我帮您跟进一下.

**Reply by MaxMKMax** [reply]
@bhu1987 您好. 请您再一次提供我支付单据和账号邮箱, 我帮您跟进一下.

**Reply by MaxMKMax** [reply]
@bhu1987 刚刚帮您再次询问了. 等待回复. 有消息马上通知您!

**Reply by bhu1987** [reply]
收到退款了，感谢。@Mike777Khoo

**Reply by MaxMKMax** [reply]
@bhu1987 您好. 谢谢您曾经支持蓝灯. 以后您需要蓝灯的时候请您谁是回来用我们的服务. 真心欢迎您回来!
